docs/source/conf.py

```diff
@@ -17,6 +17,7 @@
 
 # -- Project information -----------------------------------------------------
 
+
```

Comment:
This seems perhaps unintentional — I think this file is autogenerated by some tool, so let's not edit it spuriously if we can avoid it.
```suggestion
```